src/translation/translation_de_DE.ts

```diff
@@ -3546,12 +3546,12 @@ Wir haben Deinen Kanal stummgeschaltet und die Funktion &apos;Stummschalten&apos
     <message>
         <location filename="../serverdlg.cpp" line="63"/>
         <source>Or select &apos;%1&apos; and specify a Custom Directory address on the Options tab to register with a custom directory.</source>
```

Comment:
Note: English should also use "&quot;" (single quoted speech mark) rather than "&apos;" (apostrophe of omission).